web-client/public/js/warnings.js

```diff
@@ -79,15 +68,19 @@ export var displayPPINodeColorWarning = function (warningDisplayed) {
         ].join(" ")
     );
 
+    showWarningsModel();
+};
+
+var showWarningsModel = function () {
```

Comment:
Minor thing, won’t block merging: we actually mean the _modal_ dialog box here right? So I think the spelling on this function should be:
```suggestion
var showWarningsModal = function () {
```